src/Equations/RunawayFluid.cpp

```diff
@@ -866,6 +866,28 @@ real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRate(len_t deriv
     return dGamma;
 }
 
+/**
+ * Same as the function above, but for the formula published by Hesslow et al NF 2019
+ */
+real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRateAlt(len_t derivId) {
+    real_t *dGamma = new real_t[nr];
+    if( !( (derivId==id_Eterm) || (derivId==id_ntot) ) )
+        for(len_t ir = 0; ir<nr; ir++)
+            dGamma[ir] = 0;
+
+    // set dGamma to d(Gamma)/d(E_term)
+    for(len_t ir=0; ir<nr; ir++)
+        dGamma[ir] = avalancheGrowthRateAlt[ir] / ( Eterm[ir] - effectiveCriticalField[ir] );
```

Comment:
Is there a ``if (Eterm[ir] > effectiveCriticalField[ir]) ...`` missing here? If this method is called even if E is too low, then this would just output nonsense? And if E = Eceff DREAM would crash as it currently stands.

Reply:
avalancheGrowth.. explicitly (or well, quite implicitly actually, but it’s there) has a factor of (E-Eceff) that this is intended to cancel. This is supposed to be applied for all E. Because of the way the calculation is structured, it is a bit tricky to extract the resulting factor explicitly. At least we should handle the special case E=Eceff somehow (just set to 0? it’s quite unlikely to ever occur)

Reply:
Since we allow the avalanche growthrate to turn negative (to emulate dissipation), I don't see the issue with E<Eceff, but E=Eceff would certainly cause problems